message/html/action/like.js

```diff
@@ -24,7 +24,8 @@ exports.create = (api) => {
           }
         }
       }
-    }, when(liked, 'Unlike', 'Like'))
+    }
+    return h('span', when(liked, h('a.liked', attributes, 'Unlike'), h('a.unliked', attributes, 'Like'))
```

Comment:
I would prefer something like this:

```js
return when(liked, 
  h('a.unlike', {
    publishLike(msg, false)
  }, 'Unlike'),
  h('a.like', {
    publishLike(msg, true)
  }, 'Like'),
)
```

Also note that I'm using action verbs for the classes rather than states.